tests/services/db/test_db_response_processor.py

```diff
@@ -45,6 +45,7 @@ def test_process_events(self):
 			'format_': 'Meeting or Networking Event',
 			'organizer': 'Adam Kurtz Org',
 			'country': 'United States',
+            'admin_url': 'https://www.eventbrite.com/myevent?slow_load=1&eid=99894936444',
```

Comment:
I see this wrongly indented. I don't know if it's because of github's format

Reply:
Yes it's github's format

Reply:
Resolved